src/components/Categories/Categories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Categories(props) {
+    return (
+        <div>
+            <ExampleH1>coming soon</ExampleH1>
+        </div>
```

Comment:
Why do you need a `div` around your `h1`?

Reply:
This is only example